api/src/main/java/ai/djl/ndarray/index/NDArrayIndexer.java

```diff
@@ -50,33 +49,7 @@ public abstract class NDArrayIndexer {
      * @param index the index to get
      * @return the subarray
      */
-    public NDArray get(NDArray array, NDIndex index) {
```

Comment:
In general, I prefer leaving the definition here as an overridable default rather than moving it to MXNet because the code is generic to all NDArrayIndexers rather then being MXNet specific. It could then be reused for other engines if we wanted to dedicate more time to them

Reply:
Yeah. I have restored it.